Switch to antora.org (#46)
hercules-ci/docs.hercules-ci.com: docs/modules/ROOT/partials/snippets/BinaryCachesJson.adoc

<3> The public part of the signing key. Look for "Public Key" on your cache page (example page: `+https://mycache.cachix.org+`).
<4> A cache-specific secret key to sign store paths. You can find it in `~/.config/cachix/cachix.dhall` or your key backup after following the setup instructions on cachix.org. Make sure you copy the right key if you have multiple in your `cachix.dhall`.

// TODO: xref:reference:binary-cache-json.adoc[TODO]

> NOTE: For more detail, see https://docs.hercules-ci.com/hercules-ci/reference/binary-caches-json/ in the Reference.

Reviewer: Missing label + rephrase
```suggestion
NOTE: For more details, the Reference has a section about https://docs.hercules-ci.com/hercules-ci/reference/binary-caches-json/[binary-caches.json].
```
